pyccel/parser/semantic.py

```diff
@@ -22,7 +22,7 @@
 from pyccel.ast.basic         import PyccelAstNode, TypedAstNode, ScopedAstNode
 
 from pyccel.ast.builtins import PythonPrint, PythonTupleFunction
-from pyccel.ast.builtins import PythonComplex
+from pyccel.ast.builtins import PythonComplex, PythonDict
```

Comment:
Wouldn't it be better to organize the imports in some manner?

Reply:
Probably but in most files they are just organised by file. Do you have a suggestion for a better organisation?

Reply:
we could opt for something like this:
```
from pyccel.ast.builtins import (
    # Complex numbers 
    PythonComplex, PythonConjugate, PythonImag, PythonReal,
    
    # Data structures
    PythonDict, PythonList, PythonSet, PythonTuple, PythonTupleFunction,

    # Functionals 
    PythonEnumerate, Lambda, PythonMap, PythonPrint, PythonRange, PythonZip,
    
    # Other
    builtin_functions_dict,
)
```
This way, we won't have to repeat the file name on each line and imports will be grouped and each group will be sorted. This organization schema also reflects the role of each import and its relation to other imports from the same file.

Reply:
That looks nice in theory, but I suspect this would be a monster effort to clean up the imports in this file to always add comments about each import. I'm also not sure it's worth it for imports given that the only time I ever look at imports is when one is missing.

Let's leave this discussion here and see what Yaman thinks when he reviews

Reply:
@jalalium Do you agree that this is out of scope for this PR? If you think it is worth doing at some point then please create a discussion so we remember to talk to Yaman about this

Reply:
I agree, Let's create a discussion for later